packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -366,9 +366,24 @@ export function registerLocalNetwork() {
 
 export function isNetwork(chainId: ChainId) {
   const customChains = getCustomChainsFromLocalStorage()
-  const isMainnetOrbitChain = chainId in orbitMainnets
+  const customChainIds = customChains.map(chain => chain.chainId)
+  const isCustomOrbitChain = customChainIds.includes(chainId)
+
   const isL3Local = chainId === ChainId.L3Local
-  const isTestnetOrbitChain = chainId in orbitTestnets || isL3Local
+
+  const isMainnetOrbitChain =
+    typeof getOrbitChains({
+      mainnet: true,
+      testnet: false
+    }).find(chain => chain.chainId === chainId) !== 'undefined'
```

Comment:
This is less performant than the previous implementation but I feel like we shouldn't export both orbit objects and a getter function and have less consistent codebase.

We work with small lists so using `find` here should be fine

Reply:
Ideally our ArbitrumNetwork should have a `testnet` flag. That would simplify `isNetwork` itself as well

Reply:
This has been addressed, we use a testnet flag from arbitrum-sdk now